Bugfix and some QoL

klemenvon opened the pull request:
Some changes to how we handle terminal colors internally. Fixed bug where student numbers would get cut off too far when prettifying user IDs. These changes are only tested for staff account with no quota overruns. Should be fully tested before deploying new version. Do that before accepting the PR.

klemenvon committed:
Fixed issue where student number would get cut too far.

klemenvon committed:
Changed the way we print colors. Cleaner code.

klemenvon committed:
Updated RPM spec file for pgquota.

bedroge commented on pgquota/pgquota:
```diff
@@ -1,13 +1,43 @@
 #!/usr/bin/env python3
```
```suggestion
#!/usr/bin/python3
```
Let's stick to the OS Python here, as discussed on Slack.

bedroge committed:
add spaces around operators and after commas

bedroge committed:
add some more spaces

bedroge committed:
more style fixes, e.g. spaces instead of tabs

bedroge commented on pgquota/pgquota:
```diff
@@ -1,242 +1,328 @@
+        self.raw = subprocess.check_output(['lfs', 'quota', '-g', str(self.group), str(self.directory), '-h']).decode(
+            "utf-8")
+        if debug_flag:
+            print("{}Debug Raw{}".format(colors.get('red'), colors.get('end')))
```
`colors` has to be replaced here.

bedroge commented on pgquota/pgquota:
```diff
@@ -1,242 +1,328 @@
+            print("{}Debug Raw{}".format(colors.get('red'), colors.get('end')))
+            # print(Col.red("Debug Raw"))
+            print(self.raw)
+            print("{}End of Debug Raw{}".format(colors.get('red'), colors.get('end')))
```
and here

bedroge commented on pgquota/pgquota:
```diff
@@ -1,242 +1,328 @@
+        if self.fgrace == "none":
+            self.fgrace = self.fgrace.upper()
+        if debug_flag:
+            print("{}Debug object variables{}".format(colors.get('red'), colors.get('end')))
```
and here

bedroge commented on pgquota/pgquota:
```diff
@@ -1,242 +1,328 @@
+                    self.dusage, type(self.dusage), self.dalloc, type(self.dalloc), self.dlimit, type(self.dlimit),
+                    self.fusage, type(self.fusage), self.falloc, type(self.falloc), self.flimit, type(self.flimit),
+                    self.dgrace, type(self.dgrace), self.fgrace, type(self.fgrace)))
+            print("{}End Debug ojbect variables{}".format(colors.get('red'), colors.get('end')))
```
and here too.

bedroge commented on pgquota/pgquota:
```diff
@@ -1,242 +1,328 @@
+# 	'end'  : '\033[0m' # if this isn't at the end of the string, the formatting can overflow onto more lines of text
+# }
+
+class Col():
```
I'd just name it Color?

bedroge reviewed (changes requested):
I've made some style changes, and requested some more changes.
We should also remove all the commented lines (old functionality, e.g. the `colors` dict).

klemenvon committed:
Addressed issues raised in comments to PR #10. Updated RPM specfile.

klemenvon commented:
Addressed all the issues and entirely removed old (commented out) code or reduce fluff. Updated the RPM spec file as well, so all this needs is one more good look and then we can install it.

klemenvon committed:
Grace period is not hardcoded in output anymore, no longer 7 days.

klemenvon reviewed (commented):
Made requested changes and deleted old, commented out code.

klemenvon committed:
There were more references to 7 day period. Removed them.
